auth/api/iam/session.go

```diff
@@ -39,10 +39,25 @@ type OAuthSession struct {
 	ServerState            ServerState
 	ResponseType           string
 	PresentationDefinition PresentationDefinition
-	UserDetails            UserDetails
 	VerifierDID            *did.DID
 }
 
+// UserSession is a session-bound Verifiable Credential wallet.
+type UserSession struct {
+	// TenantDID is the requesting DID when the user session was created, typically the employer's (of the user) DID.
+	// A session needs to be scoped to the tenant DID, since the session gives access to the tenant's wallet,
+	// and the user session might contain session-bound credentials (e.g. EmployeeCredential) that were issued by the tenant.
+	TenantDID did.DID    `json:"tenantDID"`
+	Wallet    UserWallet `json:"wallet"`
+}
+
+// UserWallet is a session-bound Verifiable Credential wallet.
```

Comment:
Add some info about the security level of this wallet, since it's session bound. In the case of clustering the private key will be synchronised plain text. This is OK since it doesn't really add any security, just protocol compatibility.